tests/plugins/balloontoolbar/context/_helpers/tools.js

```diff
@@ -9,6 +9,21 @@
 		 */
 		_assertToolbarVisible: function( expected, context, msg ) {
 			assert.areSame( expected, context.toolbar._view.parts.panel.isVisible(), msg || 'Toolbar visibility' );
+		},
+
+		/*
+		 * Returns a Context instance with toolbar show/hide methods stubbed.
+		 *
+		 * @param {String[]} widgetNames List of widget names to be set as `options.widgets`.
+		 * @param {CKEDITOR.editor} editor Editor instance for which balloontoolbar context should be created.
+		 * @param {String/String[]} buttons List of buttons to be set as `options.buttons`.
+		 * @returns {CKEDITOR.plugins.balloontoolbar.context}
+		 */
+		_getContextStub: function( widgetNames, editor, buttons ) {
```

Comment:
Nearly all other of our functions for testing use `editor` as the first parameter. Please update your function.